src/index.js

```diff
@@ -1,8 +1,10 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.scss";
-import Routes from "./routes";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Routes from './routes';
+import './index.scss';
+import './fontawesome-lib';
+import './svgs.scss';
```

Comment:
This import is not needed rigth?

Reply:
I thought about abstracting all the SVG related styles to `svgs.scss` and importing it once in `index.js`

Reply:
currently required or not?

Reply:
Yes, it's required.
```
.svg-inline--fa {
    cursor: pointer;
}
```

General styles for font awesome icons (svgs)